hazelcast-jet-kafka/src/main/java/com/hazelcast/jet/impl/connector/kafka/StreamKafkaP.java

```diff
@@ -154,7 +161,13 @@ public boolean complete() {
         assert !currentAssignment.isEmpty() : "No topic partitions assigned to this processor.";
 
         if (traverser == null) {
-            ConsumerRecords<Object, Object> records = consumer.poll(POLL_TIMEOUT_MS);
+            ConsumerRecords<Object, Object> records;
+            try {
+                records = consumer.poll(POLL_TIMEOUT_MS);
+            } catch (InterruptException e) {
+                commitSync();
```

Comment:
`commitSync` should be removed here. We are sure that the records just polled won't be processed.